Doc/library/os.rst

```diff
@@ -292,7 +292,10 @@ process and user.
 .. function:: getenv(key, default=None)
 
    Return the value of the environment variable *key* if it exists, or
-   *default* if it doesn't. *key*, *default* and the result are str.
+   *default* if it doesn't. *key*, *default* and the result are str. Note that
+   since :func:`getenv` uses ``os.environ``, the mapping of :func:`getenv` is
```

Comment:
```suggestion
   since :func:`getenv` uses :data:`os.environ`, the mapping of :func:`getenv` is
```
Also let's link `os.environ` at the other occurrences.